installing/installing_vsphere/ipi/installing-vsphere-installer-provisioned-network-customizations.adoc

```diff
@@ -66,6 +66,13 @@ include::modules/nw-modifying-operator-install-config.adoc[leveloffset=+1]
 include::modules/nw-operator-cr.adoc[leveloffset=+1]
 // end network customization
 
+// Services for an external load balancer
+include::modules/nw-osp-services-external-load-balancer.adoc[leveloffset=+1]
+
+// Configuring an external load balancer
```

Comment:
Does this need to say "Configuring a user-managed load balancer" instead, so it's consistent throughout?